test/lib/rc.spec.js

```diff
@@ -55,5 +55,98 @@ describe('RC', function() {
         expect(subject.hasProjects).to.be.true;
       });
     });
+
```

Comment:
Actually, it doesn't -- the logic of `save()` is that it only persists to disk if a `path` is set in the constructor, which these tests don't do. `save()` just returns false otherwise.